src/dynarec/rv64/dynarec_rv64_00_2.c

```diff
@@ -547,7 +547,7 @@ uintptr_t dynarec64_00_2(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
         case 0x9F:
             INST_NAME("LAHF");
             READFLAGS(X_CF | X_PF | X_AF | X_ZF | X_SF);
-            ANDI(x1, xFlags, 0xFF);
```

Comment:
well, that OF2 flags doesn't seems handled in this opcode?

Reply:
OF2 has taken the place of res3 but it is not read here, so the mask below will strip the extra OF2, as I confirmed in the cosim.

Reply:
ah yes, OF is to high to get there. All good then :)